examples/distributed/main.py

```diff
@@ -177,11 +172,10 @@ def test():
         with torch.no_grad():
             if args.cuda:
                 data, target = data.cuda(), target.cuda()
-            data, target = Variable(data), Variable(target)
             output = model(data)
             test_loss += to_python_float(F.nll_loss(output, target, size_average=False).data) # sum up batch loss
             pred = output.data.max(1, keepdim=True)[1] # get the index of the max log-probability
-            correct += pred.eq(target.data.view_as(pred)).cpu().sum()
+            correct += pred.eq(target.data.view_as(pred)).cpu().float().sum()
```

Comment:
Half<->float conversions are generally super slow on the CPU.  Can we change this to `.float().cpu()`?

Reply:
Sure! Good to know it's slower on the CPU.